### PR TITLE
Bone Regeneration Fix

### PR DESCRIPTION
**Pull request by Geevies**
* Bones no longer spontaneously repair themselves.

Fixes https://github.com/Aurorastation/Aurora.3/issues/11569

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: code/modules/mob/living/carbon/human/human_damage.dm [hunk 1]
```diff
@@ -281,7 +281,7 @@
 	if(!length(parts))
 		return
 	var/obj/item/organ/external/picked = pick(parts)
-	if(picked.heal_damage(brute, burn, prosthetic))
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/human_damage.dm:284]: Don't use `null` here, do `robo_repair = prosthetic`.

**Review by NonQueueingMatt — DISMISSED** [review]

File: code/modules/mob/living/carbon/human/human_damage.dm [hunk 2]
```diff
@@ -281,7 +281,7 @@
...
+	if(picked.heal_damage(brute, burn, null, prosthetic))
```
Comment (Geevies) [inline code/modules/mob/living/carbon/human/human_damage.dm:284]:
```suggestion
	if(picked.heal_damage(brute, burn, robo_repair = prosthetic))
```

**Review by Alberyk — APPROVED** [review]

**Review by Doxxmedearly — APPROVED** [review]

**Pull request merged**